src/DecileCounter.java

```diff
@@ -0,0 +1,50 @@
+import java.util.*;
+
+public class DecileCounter {
+    public static void Decile() {
+        Scanner inputScanner = new Scanner(System.in);
+        List<Integer> numbers = new ArrayList<>();
+        int num;
+        do {
+            System.out.print("Enter a number from 1 to 1000 (or 'END' to stop): ");
+            String str = inputScanner.nextLine();
+            if (str.equals("END")) {
+                break;
+            }
+            try {
+                num = Integer.parseInt(str);
+                if (num < 1 || num > 1000) {
+                    System.out.println("Invalid input. Number must be between 1 and 1000.");
+                    continue;
+                }
+                numbers.add(num);
+            } catch (NumberFormatException e) {
+                System.out.println("Invalid input. Please enter a number or 'END'.");
+            }
+        } while (true);
+
+        Collections.sort(numbers);
+        int size = numbers.size();
+        int thirdDecileIndex = (int) Math.ceil(size * 0.3) - 1;
+        int sixthDecileIndex = (int) Math.ceil(size * 0.6) - 1;
+        int thirdDecileCount = 0;
+        int sixthDecileCount = 0;
+
+        for (int i = 0; i < size; i++) {
```

Comment:
this don't meet business requirements. Read what is decile https://en.wikipedia.org/wiki/Decile and set for all population (1-1000)